journal/202205.md

```diff
@@ -0,0 +1,458 @@
+
+}
+
+Contents: { _Sync ToC at the very end, when all text is finished_ }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+{ _Add individual sections for high-impact announcements before Development, such as consensus votes, software releases, milestones, major bugs, etc._ }
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+_dcrd is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+[v1.7.2 patch](https://github.com/decred/dcrd/releases/tag/release-v1.7.2) has been released fixing a rare and hard to hit case when optional indexing is enabled.
+
+The general direction in May was to further leverage the consensus changes that activated recently, by adding optimizations and removing unneeded code.
+
+- Optimized [stake node pruning](https://github.com/decred/dcrd/pull/2943) by using block headers instead of checkpoints (reducing dependency on the latter). Header-based sync operations are more efficient and pave the way to multi-peer block downloads.
+- Header proof logic has been [consolidated](https://github.com/decred/dcrd/pull/2937), resulting in the p2p and rpcserver handlers no longer needing to understand anything about the proofs. { _maybe remove as minor_ }
+- Retroactively fixed revocation [fee limit bug](https://github.com/decred/dcrd/pull/2948) that allowed one party to increase the fee paid to miners for another party in split transactions. This has never been exploited on `mainnet` and _can not_ be exploited since the automatic revocations agenda has activated.
+- Reworked [old block rejection](https://github.com/decred/dcrd/pull/2945) logic to replace the use of checkpoints with the `assumevalid` block (hard-coded in each release). Semantics have been clarified to reflect that "checkpoints" are now only used for handling old forks and are no longer used for optimizations (that now rely on other methods). The `--nocheckpoints` CLI option has been replaced with `--allowoldforks`.
+  - { re /byebye checkpoints in #dev - is this a final defeat of checkpoints or is there more to do? }
+- Removed the deprecated [address index](https://github.com/decred/dcrd/pull/2930) along with the associated CLI flags (`--addrindex` and `--dropaddrindex`) and `searchrawtransactions` JSON-RPC.
+  - { _Is there more info on why it was deprecated and removed, and what replaces it?_ }
```

Comment:
It's a lot of overhead from a development perspective to maintain, it isn't used by anything of note, and it doesn't really easily provide what most people want from addresses anyway, which is a full balance as opposed to all of the individual transactions.  All of the information it provided, and more, is available via dcrdata.

Reply:
Great summary, added almost as is.